Dockerfile

```diff
@@ -18,9 +18,17 @@ RUN adduser --disabled-password --gecos "" mitodl
 RUN mkdir /var/media && chown -R mitodl:mitodl /var/media
 
 # Install project packages
-COPY requirements.txt /tmp/requirements.txt
-COPY test_requirements.txt /tmp/test_requirements.txt
-RUN pip install -r requirements.txt -r test_requirements.txt
+ENV  \
+  # poetry:
+  POETRY_VERSION=1.5.1 \
+  POETRY_VIRTUALENVS_CREATE=false \
+  POETRY_CACHE_DIR='/tmp/cache/poetry'
+
+# Install poetry
+RUN pip install "poetry==$POETRY_VERSION"
```

Comment:
As discussed and mentioned in the Poetry docs, we should install Poetry using `curl -sSL https://install.python-poetry.org | python3 -` and use their install script instead of installing from pip to avoid modifying the system Python.